### ISSUE TITLE
build against an old-enough glib 

### ISSUE THREAD

**Issue by alsuren** [alsuren/cargo-quickinstall#50]
centos 6 might do, according to https://www.reddit.com/r/rust/comments/k9wiff/introducing_cargoquickinstall/gf7wkw2/ , but check what `rustup` does before starting work. I expect that there will be crates that don't build on centos 6 that do build on macos and ubuntu 20.04. Would be nice to have a scheme that works for everyone

**Comment by NobodyXu** [comment]
https://crates.io/crates/cargo-zigbuild uses zig-cc, which can cross compile any rust project to targets with any glibc version.
